Android.bp

```diff
@@ -14623,6 +14623,7 @@ genrule {
         "src/trace_processor/perfetto_sql/stdlib/stack_trace/jit.sql",
         "src/trace_processor/perfetto_sql/stdlib/stacks/cpu_profiling.sql",
         "src/trace_processor/perfetto_sql/stdlib/time/conversion.sql",
+        "src/trace_processor/perfetto_sql/stdlib/trace/stats.sql",
```

Comment:
Can we please call the package traced to make it a little more obvious that this only for traces collected using Perfetto tools.

Reply:
Good idea, done